roles/nova-common/templates/etc/serverspec/nova_spec.rb

```diff
@@ -55,10 +55,9 @@
 
 describe file('/etc/nova/nova.conf') do
   it { should contain 'debug = {{ nova.logging.debug }}' }
-end 
+end
 
-describe file( '{{ nova.state_path }}/instances' ) do
-  it { should exist }
+has_file = file('{{ nova.state_path }}/instances').exists?
+describe file( '{{ nova.state_path }}/instances' ), :if => has_file do
   it { should be a directory }
```

Comment:
(this does not block the PR)

Is there not a tech spec on the ownership / permissions of the instances directory?